koku/reporting/provider/azure/models.py

```diff
@@ -61,6 +61,7 @@ class Meta:
     service_name = models.CharField(max_length=50, null=False)
     service_info1 = models.TextField(null=True)
     service_info2 = models.TextField(null=True)
+    instance_type = models.CharField(max_length=50, null=True)
```

Comment:
With this model change we can probably update the summary SQL here https://github.com/project-koku/koku/blob/8aaee11d1b0e33711660f0202f333440079a3acc/koku/masu/database/sql/reporting_azurecostentrylineitem_daily_summary.sql#L9

Also will need an initial migration change for the existing database to populate this field.